### DISCUSSION TITLE
When Build generation is triggered, nothing happens for a few hrs, and just after generation process is started

### DISCUSSION THREAD

**Discussion by vmboiko** [codemagic-ci-cd/codemagic-docs#2449] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

All need to do is trigger Build generation process

### Expected results

No need to have big delay for few hrs

### Actual results

Always big delay before building starts
Status **queued** is for a few hrs, and 

Img below is what I see for a few hrs:

<img width="603" alt="image" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/104016736/89e8c9ab-fb10-4679-849a-7e971db0861a">

**Comment by mikhail-tokarev** [comment]
hey! please check status.codemagic.io page and this thread for more information https://github.com/orgs/codemagic-ci-cd/discussions/2424
